src/commands/start.rs

```diff
@@ -40,6 +40,16 @@ impl StartCommand {
             process::exit(1);
         });
 
+        if let Some(config) = &APP.config().metrics {
+            let address = config.bind_address.clone();
+            info!("Starting up prometheus server on {}", address);
```

Comment:
ehh, 1/5 i would maybe call it

```suggestion
            info!("Starting up metrics exporter on {}", address);
```

Reply:
or `prometheus exporter` like it says below